### PR TITLE
best attempt to get to design of new import button

### LINKED ISSUE #1792: Change Import resource template / profile button
To match:
image

If it cannot easily be placed adjacent to search box, placing it slightly above is acceptable.

### PR DESCRIPTION
**Pull request by peetucket**
(sort of) fixes #1792 

I was able to more or less get the search bar and import button on the same line, but if you shrink your screen down, the search box overruns the import button (due to a fixed pixel setting for the width of the search box).  If you do the right thing and make the search box width = 100%, then it comes out too small to show the placeholder text.  I can't figure out how to get the search bar and the import button to make better use of the space available otherwise.

### CODE REVIEW

File: src/components/templates/ImportFileZone.jsx [hunk 1]
```diff
@@ -209,9 +210,11 @@ class ImportFileZone extends Component {
     return (
       <section>
         <div className="ImportFileZone" style={importFileZone}>
+          <FontAwesomeIcon icon={uploadIcon} />
           <button id="ImportProfile"
-                  className="btn btn-primary btn-lg"
-                  onClick={this.handleClick}>Import a Profile or Resource Template</button>
+                  className="btn-link"
```
Comment (justinlittman) [inline src/components/templates/ImportFileZone.jsx]: Should be `"btn btn-link"`.

File: src/components/templates/ImportResourceTemplate.jsx [hunk 2]
```diff
@@ -13,9 +13,17 @@ const ImportResourceTemplate = props => (
   <section id="importResourceTemplate">
     <UpdateResourceModal />
     <Header triggerEditorMenu={props.triggerHandleOffsetMenu}/>
-    <ImportFileZone />
+    <div className="container">
```
[the remark this answers is not in the document]
Comment (peetucket) [reply src/components/templates/ImportResourceTemplate.jsx]: reverted

File: src/components/templates/TemplateSearch.jsx [hunk 3]
```diff
@@ -41,8 +41,8 @@ const TemplateSearch = () => {
         <div className="col">
           <form className="form-inline" onSubmit={event => event.preventDefault()}>
             <div className="form-group" style={{ paddingBottom: '10px', paddingTop: '10px' }}>
-              <label htmlFor="searchInput">Find a resource template</label>&nbsp;
-              <div className="input-group" style={{ width: '750px', paddingLeft: '5px' }}>
+              <label className="font-weight-bold" htmlFor="searchInput">Find a resource template</label>&nbsp;
+              <div className="input-group" style={{ width: '500px', paddingLeft: '5px' }}>
```
Comment (justinlittman) [inline src/components/templates/TemplateSearch.jsx]: Can you set this back to 750px?

File: src/components/templates/ImportFileZone.jsx [hunk 4]
```diff
@@ -209,9 +210,11 @@ class ImportFileZone extends Component {
     return (
       <section>
         <div className="ImportFileZone" style={importFileZone}>
+          <FontAwesomeIcon icon={faUpload} />
```
Comment (justinlittman) [inline src/components/templates/ImportFileZone.jsx]: Sorry to be fiddly, but can you adjust the icon alignment?
image

File: src/components/templates/TemplateSearch.jsx (hunk 3 above)
Comment (peetucket) [reply src/components/templates/TemplateSearch.jsx]: yes, though i think its better to not force it that wide (when you make the viewport smaller, it looks worse)

File: src/components/templates/ImportFileZone.jsx (hunk 4 above)
Comment (peetucket) [reply src/components/templates/ImportFileZone.jsx]: yeah, that bugged me too, but i had a hard time doing it in CSS...easy fix is to make it part of the link, but that makes the icon blue and underlines it like this:

Screen Shot 2019-11-20 at 10 24 02 AM

Comment (peetucket) [reply src/components/templates/ImportFileZone.jsx]: never mind - aligned correctly

**Pull request merged**